.commitlintrc.yml

```diff
@@ -1,2 +1,4 @@
 extends:
   - '@commitlint/config-conventional'
+rules:
+  header-max-length: [1, 'always', 150]
```

Comment:
What is this for?

Reply:
It's commit message maximum length. I had one lengthy commit which I could not commit and it think 100 characters is too few considering that we also use our commit message for CHANGELOG.